datasets/BNCIsujet1.py

```diff
@@ -1,39 +1,48 @@
 from benchopt import BaseDataset, safe_import_context
+import pickle 
 
 
 # Protect the import with `safe_import_context()`. This allows:
 # - skipping import to speed up autocompletion in CLI.
 # - getting requirements info when all dependencies are not installed.
 with safe_import_context() as import_ctx:
     import numpy as np
+    import moabb
+    from moabb.datasets import BNCI2014001
+    from moabb.evaluations import CrossSessionEvaluation
```

Comment:
Should move to `objective.py`

As discussed IRL, we should probably do the split in the objective, and simply return the dataset from here?